serahinWeb/api/api.nl.yml

```
@@ -4,185 +4,185 @@ nl:
-        create_green_card: Ajouter une carte verte
-        delete: Supprimer
-        mail: Générer un mail
-        reject_for_bad_customer: Rejeter pour mauvais client
-        reject_for_bad_input: Retour a l'upload des documents
-        renew_green_card: Renouveler la carte verte
-        reset_password: Envoyer mail reset MDP
-        resume_purchase: Reprendre l'achat
-        stop: Résilier / Suspendre / Transfert en défaveur
-        stopping_date: Date à laquelle le contrat sera résilié.
-        stopping_reason: Quelle en est la raison ?
-        upsell: Upseller le contrat
-        validate: Valider documents uploadés
-        validate_amendment: Valider l'avenant
-        verify: Approuver contrat (non-fiché)
+        archive: Archiveren
+        become: Inloggen
+        couple: Het contract koppelen
+        create_amendment: Een nieuw aanhangsel aanmaken
+        create_contract: Het contract valideren en creëren
+        create_green_card: Een groene kaart toevoegen
+        delete: Verwijderen
+        mail: Een e-mail genereren
+        reject_for_bad_customer: Afwijzen wegens slechte klant
+        reject_for_bad_input: Terug naar de upload van documenten CONTEXT
+        renew_green_card: De groene kaart vernieuwen
+        reset_password: Mail verzenden reset ww
+        resume_purchase: De aankoop hervatten
+        stop: Opzeggen / Opschorten / Transfer in nadeel
+        stopping_date: Datum waarop het contract opgezegd wordt.
+        stopping_reason: Wat is de reden ?
+        upsell: Het contract verzegelen CONTEXT
+        validate: De geüploade documenten valideren
+        validate_amendment: Het aanhangsel valideren
+        verify: Contract goedkeuren (non-fiché) CONTEXT blacklisted or non blacklisted?
       tooltip:
-        stopping_date: Mettre la date d'aujourd'hui pour résiliation/suspension immédiate
+        stopping_date: Vandaag dateren voor onmiddellijke opzegging/opschorting
     belongings:
       messages:
         errors:
           destroy:
-            contractually_frozen: L'objet de risque est lié à un contrat
+            contractually_frozen: Het risico-object is gelinkt aan een contract
         success:
-          create_or_update: Objet de risque créé ou modifié avec succès
-          destroy: Objet de risque supprimé avec succès
+          create_or_update: Risico-object succesvol gecreëerd of gewijzigd
+          destroy: Risico-object succesvol verwijderd
     car_brands_and_models:
       messages:
         success:
-          index: Marques et modèles de voiture récupérés avec succès
+          index: Merken en modellen van de auto succesvol hersteld
     cars:
       messages:
         errors:
           update:
-            brand_and_model: Cette marque et/ou modèle n'existent pas. Contacter les techies s'ils doivent être rajoutés
+            brand_and_model: Dit merk en/of model bestaat niet. Contacteer de techies dat ze deze moeten vervangen CONTEX TECHIES
```

techies ?c'est comme les techniciens ?
```suggestion
            brand_and_model: Dit merk en/of model bestaat niet. Contacteer de techies dat ze deze moeten vervangen CONTEXT TECHIES
```

---

```suggestion
            brand_and_model: Dit merk en/of model bestaat niet. Contacteer het IT-team dat ze deze moeten vervangen 
```